api/src/main/java/io/strimzi/api/kafka/model/template/ZookeeperClusterTemplate.java

```diff
@@ -111,7 +112,10 @@ public void setZookeeperContainer(ContainerTemplate zookeeperContainer) {
         this.zookeeperContainer = zookeeperContainer;
     }
 
-    @Description("Template for the Kafka broker TLS sidecar container")
+    @DeprecatedProperty
+    @Deprecated
+    @Description("Template for the Kafka broker TLS sidecar container. " +
```

Comment:
I know it doesn't matter as this is now deprecated, but isn't this technically the `ZooKeeper TLS sidecar container`?

Or do you refer to it as the kafka broker sidecar as its job is to speak to the kafka broker?